packages/embed/src/factories/create-widget/create-widget.ts

```diff
@@ -66,22 +75,31 @@ export const createWidget = (formId: string, options: WidgetOptions): Widget =>
   }
 
   if (widgetOptions.enableFullscreen) {
+    let backgroundColor = ''
     const { container } = options
     const autoResize = makeAutoResize(container)
     const originalHeight = container.style.height
     const openPopup = () => {
       container.classList.add('tf-v1-widget-fullscreen')
+      if (options.opacity !== undefined) {
```

Comment:
and what if `null`?

Reply:
It can be `undefined` or integer, [see transformInteger method](https://github.com/Typeform/embed/blob/dist1607/packages/embed/src/utils/load-options-from-attributes.ts#L31-L34).

Reply:
💯